app/kumactl/cmd/completion/completion_test.go

```diff
@@ -50,7 +50,7 @@ var _ = Describe("kumactl completion", func() {
 			// when
 			actual := stdout.Bytes()
 			// then
-			Expect(len(actual)).To(Equal(len(expected)))
+			Expect(len(actual)).To(Equal(len(expected)), "run kumactl generate bash/fish/zsh | pbcopy and override testdata/*.golden")
```

Comment:
This is OSX biased advice.

Reply:
how so? `pbcopy` is also available in linux